src/brasil/gov/agenda/browser/templates/agendaview.pt

```diff
@@ -9,7 +9,18 @@
         tal:content="string:${context/Title} ${context/autoridade}"></h1>
   </metal:slot>
   <metal:block fill-slot="content-core">
-    <tal:semagendadiaria tal:condition="not:view/editable">
+    <tal:canedit tal:condition="view/editable" tal:define="batch view/results">
```

Comment:
I prefer to add an `is_anonymous()` function to do this; that way we can clean up the code a little bit.

Reply:
and if you are not anonymous but don't have permission to edit?